src/main/java/io/apimatic/core/types/http/response/Dynamic.java

```diff
@@ -138,7 +135,6 @@ public double parseAsDouble() throws ParseException {
      * Parse response as string.
      * 
      * @return Parsed value
```

Comment:
ParseException description should also be reverted in the method summary.

Reply:
Reverted